org.apache.directory.Studio.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.apache.directory.Studio
+
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+command: /app/ApacheDirectoryStudio/ApacheDirectoryStudio
+finish-args:
+  - --device=dri
+  - --device=shm
+  - --env=PATH=/app/jre/bin:/usr/bin
+  - --persist=.ApacheDirectoryStudio
+  - --persist=.eclipse
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: ApacheDirectoryStudio
+    buildsystem: simple
+    build-commands:
+      - mv ApacheDirectoryStudio /app
+    sources:
+      - type: archive
+        url: https://dlcdn.apache.org/directory/studio/2.0.0.v20210717-M17/ApacheDirectoryStudio-2.0.0.v20210717-M17-linux.gtk.x86_64.tar.gz
+        sha256: 24764eff94ad3a257a899d34f9933276af0d201fba9017d6737558b2ffd5eda7
+        strip-components: 0
+        x-checker-data:
+          type: html
+          url: https://directory.apache.org/studio/download/download-linux.html
+          pattern: |
+            <a href="(https://dlcdn.apache.org/directory/studio/([^/]+)/ApacheDirectoryStudio-[^"]+-linux.gtk.x86_64.tar.gz)">
+
+  - name: ApacheDirectoryStudio
+    buildsystem: simple
+    build-commands:
+      - install -Dt /app/share/applications org.apache.directory.Studio.desktop
+      - install -Dt /app/share/metainfo org.apache.directory.Studio.metainfo.xml
+      - install -D studio-icon_128x128.png /app/share/icons/hicolor/128x128/apps/org.apache.directory.Studio.png
+    sources:
+      - type: file
+        path: org.apache.directory.Studio.desktop
```

Comment:
so Apache ship pre builds for Linux and don't even provide a .desktop file nor an icon?

Reply:
I couldn't find a `.desktop` file in the bundle.

There is a 48×48 XPM at the top level of the bundle… Oh, there actually is a 128×128 PNG buried in there.